server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/LoadFiles.java

```diff
@@ -219,7 +221,8 @@ void load(List<TabletMetadata> tablets, Files files) {
           server = location.getHostAndPort();
         }
 
-        Set<TabletFile> loadedFiles = tablet.getLoaded().keySet();
+        Set<TabletFile> loadedFiles = tablet.getLoaded().keySet().stream()
```

Comment:
Before this change, this code it may have been comparing StoredTabletFile to TabletFile later in the code.  I think this was a place that was relying on the sketchy hashCode and equals situation.

Reply:
I think you are right so this help things be more clear now that we don't need to rely on that sketchy equals anymore.

Before the change, `StoredTabletFile` always just delegated to the `TabletFile` for equals/hashcode (which uses the normalized path for comparison). Now, since we are mapping the set of StoredTabletFiles to a set of TabletFiles, it will just always compare TabletFile to TabletFile which will still use the normalized path as before so the equals is the same but we don't have to rely on StoredTabletFile not implementing a proper equals.